modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,173 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, {SMProductName} (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like {product-title} monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with user-workload monitoring.
+
+.Prerequisites
+
+* You have cluster administrator permissions.
```

Comment:
I would remove this prerequisite, because this is only needed to have an access to `openshift-monitoring` namespace, so I would prefer to list the following requirements:
1) UWM monitoring is enabled.
2) OSSM Operator 2.4 is installed.

```suggestion
* You have cluster administrator permissions.
```